cli/src/program_v4.rs

```diff
@@ -1051,14 +1067,24 @@ fn send_messages(
     let send_message = |message: Message, signers: &[&dyn Signer]| {
```

Comment:
`send_message` is a bit confusing, since it could also be just signing, and not sending.

Reply:
Well, even with offline signing it is kind off sending it to the hardware wallet instead of the network, right?
Anyway, if you have a suggestion I am all ears.